### ISSUE TITLE
How can i change ~ from my promp

### ISSUE THREAD

**Issue by 2Vim** [romkatv/powerlevel10k#2705]
I want to remove the ~ from my prompt from my .p10k, I have tried many things but I can't, can anyone give me a solution?

![image](https://github.com/romkatv/powerlevel10k/assets/172663851/e862c743-ee2b-4b69-a00d-a11183c1d7e9)

**Comment by romkatv** [comment]
Do you know what it means?

**Comment by 2Vim** [comment]
Yes, I know its the user's home directory, but I have seen in other prompt's that the ~ does not exist

**Comment by Syphdias** [comment]
So you want not directory segment?
Go to your `.p10k.zsh` file and search for `POWERLEVEL9K_LEFT_PROMPT_ELEMENTS` and remove the line containing `dir`, you could also comment it out by prefixing it with `#`.
